spec/chromium-spec.js

```diff
@@ -281,15 +281,14 @@ describe('chromium feature', () => {
         expect(event.data).to.equal('undefined undefined undefined undefined');
       });
 
-      // FIXME: disabled during chromium update due to crash in content::WorkerScriptFetchInitiator::CreateScriptLoaderOnIO
-      xit('has node integration with nodeIntegrationInWorker', async () => {
+      it('has node integration with nodeIntegrationInWorker', async () => {
         const webview = new WebView();
         webview.addEventListener('console-message', (e) => {
           console.log(e);
         });
         const eventPromise = waitForEvent(webview, 'ipc-message');
         webview.src = `file://${fixtures}/pages/shared_worker.html`;
-        webview.setAttribute('webpreferences', 'nodeIntegration, nodeIntegrationInWorker');
+        webview.setAttribute('webpreferences', 'nodeIntegration, nodeIntegrationInWorker, contextIsolation=no');
```

Comment:
Good opportunity to refactor the test to work with context isolation https://github.com/electron/electron/blob/main/spec/fixtures/pages/shared_worker.html :)